group_vars/all.yml

```diff
@@ -1,7 +1,13 @@
 ---
+common_repository:
+  base_url: "https://artifacts.alfresco.com/nexus/content"
+  repository_community: "groups/public"
+  repository_enterprise: "groups/private"
+  group_id: "org/alfresco"
+
```

Comment:
What about having a yaml structure as follow:

```yaml
artifacts_repositories:
  - id: enterprise
    base_url: ....
  - id: community
    base_url: .....
``` 

This would allow for customers to provide additional repos of their own by adding an entry to this list and re-use our logic to download artifacts